examples/full/main.tf

```
@@ -110,6 +110,7 @@ module "control_plane" {
   licence_key_ps_arn                  = var.licence_key_ps_arn
   enable_verbose_logging              = var.enable_verbose_logging
   grant_assume_on_role_arns           = var.control_plane_grant_assume_on_role_arns
+  alb_security_group_arn              = module.alb.alb_security_group_id
```

variable should be named id not arn
```suggestion
  alb_security_group_id             = module.alb.alb_security_group_id
```